interface/fax/fax_dispatch.php

```diff
@@ -614,203 +594,215 @@ function allCheckboxes(issel) {
-     title='Your comments to include with this message' /></textarea>
-   </td>
-  </tr>
-  <tr>
-   <td class='itemtitle' nowrap><?php echo xlt('Quality'); ?></td>
-   <td>
-    <input type='radio' name='form_finemode' value='' /><?php echo xlt('Normal'); ?> &nbsp;
-    <input type='radio' name='form_finemode' value='1' checked /><?php echo xlt('Fine'); ?> &nbsp;
-   </td>
-  </tr>
- </table>
-</div><!-- end div_forward -->
-
-<p><b><?php echo xlt('Delete Pages'); ?>:</b>&nbsp;
+<span class="font-weight-bold"><?php echo xlt('Forward Pages via Fax'); ?></span></p>
+
+<!-- Forward Pages via Fax Section -->
+<div id='div_forward' class='jumbotron' style='display:none;'>
+    <!-- From Section -->
+    <div class="form-row mt-2">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('From'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_from' class='form-control' data-toggle='tooltip' data-placement='top' title='Type your name here'>
+        </div>
+    </div>
+    <!-- To Section -->
+    <div class="form-row mt-2">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('To{{Destination}}'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_to' class='form-control' data-toggle='tooltip' data-placement='top' title='Type the recipient name here'>
+        </div>
+    </div>
+    <!-- Fax Section -->
+    <div class="form-row mt-2">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Fax'); ?></label>
+        <div class="col-10">
+            <input type='text' size='10' name='form_fax' class='form-control' data-toggle='tooltip' data-placement='top' title='The fax phone number to send this to'>
+        </div>
+    </div>
+    <!-- Message Section -->
+    <div class="form-row mt-2">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Message'); ?></label>
+        <div class="col-10">
+            <textarea name='form_message' rows='3' cols='30' class='form-control'
+                data-toggle='tooltip' data-placement='top'
+                title='Your comments to include with this message'>
+            </textarea>
+        </div>
+    </div>
+    <!-- Quality Section -->
+    <div class="form-row mt-2">
+        <label class="col-2 col-form-label font-weight-bold"><?php echo xlt('Quality'); ?></label>
+        <div class="col-10">
+            <input type='radio' name='form_finemode' value=''><?php echo xlt('Normal'); ?>
+            <input type='radio' name='form_finemode' value='1' checked><?php echo xlt('Fine'); ?>
+        </div>
+    </div>
+</div>
+
+<p><span class="font-weight-bold"><?php echo xlt('Delete Pages'); ?>:</span>&nbsp;
 <input type='radio' name='form_cb_delete' value='2' />All&nbsp;
 <input type='radio' name='form_cb_delete' value='1' checked />Selected&nbsp;
 <input type='radio' name='form_cb_delete' value='0' />None
 </p>
```

Comment:
I tried to change to the Bootstrap form-check style. But it seems to overflow occurred in `oe-bootstrap` style.

![image](https://user-images.githubusercontent.com/22230889/83408551-60166300-a445-11ea-8171-8b3995bb8d6a.png)

Reply:
Crappppppppppp --- @stu01509 go to `interface/themes/oe-bootstrap.scss` and remove the `input[type=checkbox], input[type=radio]` line that incorporates this and test the form elements out globally - I should've removed this line. Also remove the `.form-group` line as well as this shouldn't have existed.